tensorflow_io/__init__.py

```diff
@@ -1 +1,52 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""tensorflow-io"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
 
+import os
+import sys
+import inspect
+
+from tensorflow.python.framework import errors_impl
+from tensorflow.python.framework import load_library
+from tensorflow.python.platform import resource_loader
+
+# NOTE: __datapath__ is a hidden value used by test,
+# in case .so files are on a different path.
+__datapath__ = None
```

Comment:
Why would `.so` files be on different paths?

Reply:
This is to allow us to test two scenarios:
1) Install package from whl files. In this situation, `.so` file will be part of the path so `__datapath__` will always be None, no change needs.
2) Not install the package (in development). In this situation, as we split the C++ build to use bazel and pytest for python, the `.so` files are generated in `bazel-bin` and the python code are in the original source tree. However, bazel does not allow us to write `.so` file into the original source tree. So `__datapath__` is added to capture `bazel-bin`.

This is mainly for easy local development:
```
$ bazel build .....
$ python -m pytest tests --binary bazel-bin -s
```
The above will be all we need for local development and debugging. No installation required.